### PR TITLE
ChromeDriver improvement

### PR DESCRIPTION
**Pull request by aweng98**
Workaround issue `WebDriverException unknown error: DevToolsActivePort file doesn't exist`

**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

### CODE REVIEW

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/fixture/WorkspaceFixtures.scala [hunk 1]
```diff
@@ -33,7 +33,8 @@ trait WorkspaceFixtures extends ExceptionHandling with RandomUtil { self: TestSu
                    (testCode: (String) => Any)(implicit token: AuthToken): Unit = {
     val workspaceName = uuidWithPrefix(namePrefix, " ")
     Orchestration.workspaces.create(namespace, workspaceName, authDomain)
-    Orchestration.workspaces.updateAcl(namespace, workspaceName, aclEntries)
+    if (aclEntries.nonEmpty)
+      Orchestration.workspaces.updateAcl(namespace, workspaceName, aclEntries)
```
Comment (ghost) [inline serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/fixture/WorkspaceFixtures.scala:37]: nit - this is out of scope, would you mind commenting why it's needed?

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/test/WebBrowserSpec.scala [hunk 2]
```diff
@@ -74,6 +74,15 @@ trait WebBrowserSpec extends WebBrowserUtil with ExceptionHandling with LazyLogg
...
+    options.addArguments("--disable-default-apps")
+    options.addArguments("--disable-gpu")
+    options.addArguments("--no-first-run")
+    options.addArguments("--enable-automation")
+    options.addArguments("--test-type=webdriver")
+    options.addArguments("--disable-dev-shm-usage")
+    options.addArguments("--disable-hang-monitor")
+    options.addArguments("--disable-popup-blocking")
+    options.addArguments("--disable-sync")
     if (java.lang.Boolean.parseBoolean(System.getProperty("burp.proxy"))) {
```
Comment (ghost) [inline serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/test/WebBrowserSpec.scala]: have you considered adding `--disable-background-timer-throttling`?

https://hi.service-now.com/kb_view.do?sysparm_article=KB0621889

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/test/WebBrowserSpec.scala [hunk 3]
```diff
@@ -74,6 +74,17 @@ trait WebBrowserSpec extends WebBrowserUtil with ExceptionHandling with LazyLogg
...
+    options.addArguments("--disable-gpu")
+    options.addArguments("--no-first-run")
+    options.addArguments("--enable-automation")
+    options.addArguments("--test-type=webdriver")
+    options.addArguments("--disable-dev-shm-usage")
+    options.addArguments("--disable-hang-monitor")
+    options.addArguments("--disable-popup-blocking")
+    options.addArguments("--disable-sync")
+    options.addArguments("--disable-background-timer-throttling")
+    // options.addArguments("--disable-renderer-backgrounding")
```
Comment (ghost) [inline serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/test/WebBrowserSpec.scala]: I see the UI test passed. Do you think it failed due to this line? If so, will you delete the line or add a comment?

**Review by ghost — COMMENTED** [review]
A couple comments inline. I think you need to add `"TRAVIS-REPLACE-ME"` to `README.md`. I'll 👍once the relevant UI pr has passed a couple rounds of tests.

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/fixture/WorkspaceFixtures.scala (hunk 1 above)
Comment (aweng98) [reply serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/fixture/WorkspaceFixtures.scala:37]: calling api `updateAcl` when `aclEntries` is empty is not necessary. default `aclEntries` is empty in new workspace.

**Review by ghost — APPROVED** [review]
